src/network/modules/kvStore/kvStore.ts

```diff
@@ -97,17 +103,27 @@ function getKVStoreInfo(message: Message): KVStoreInfo {
   return result
 }
 
-function getKVStoreValue(message: Message): KVStoreValue {
+function getKVStoreList(message: Message): KVStoreList {
+  const data = message.getPayload()
+  const result: KVStoreList = {
+    keys: (data.get(0) as Buffer[]).map(bytes => bytes.toString()),
```

Comment:
For the sake of posterity, a key is a `bstr` which can be non-ascii/non-utf. Not blocking for demo purposes.

Reply:
Hmm... JavaScript requires Object keys to be Strings. I could use a Map instead?

Reply:
I'm not sure we want to display binary data, unless encoded in, e.g., base64. Is there a method you can use to detect is the data contains non-ascii/non-utf chars?